aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -96,21 +73,56 @@ void div_kernel(TensorIterator& iter) {
   }
 }
 
+void mul_kernel(TensorIterator& iter) {
+  if (iter.dtype() == ScalarType::Bool) {
+    cpu_kernel(iter, [=](bool a, bool b) -> bool { return a && b; });
+  } else if (isComplexType(iter.dtype())) {
+      AT_DISPATCH_COMPLEX_TYPES(iter.dtype(), "mul_cpu", [&]() {
+        cpu_kernel(iter,
+          [=](scalar_t a, scalar_t b) -> scalar_t { return a * b; });
+     });
+  } else {
+    AT_DISPATCH_ALL_TYPES_AND(kBFloat16, iter.dtype(), "mul_cpu", [&]() {
+      cpu_kernel_vec(iter,
+        [=](scalar_t a, scalar_t b) -> scalar_t { return a * b; },
+        [=](Vec256<scalar_t> a, Vec256<scalar_t> b) {
+          return a * b;
+        });
+    });
+  }
+}
+
 void logical_xor_kernel(TensorIterator& iter) {
   cpu_kernel(iter,
     [](bool a, bool b) -> bool {
       return a != b;
     });
 }
 
-} // anonymous namespace
+void remainder_kernel(TensorIterator& iter) {
+  AT_DISPATCH_FLOATING_TYPES_AND(kBFloat16, iter.dtype(), "remainder_cpu", [&]() {
+    cpu_kernel_vec(iter,
+    [=](scalar_t a, scalar_t b) -> scalar_t {
```

Comment:
nit: You probably do not need the `=`.

Reply:
For both `scalar_t` and `Vec256<scalar_t>`?